centaur/src/main/resources/standardTestCases/output_filename_interpolation/output_filename_interpolation.wdl

```diff
@@ -8,9 +8,11 @@ task output_filename_interpolation {
         echo a > ${outputMain}
         echo b > ${outputMain + "." + outputSuffix}
         echo c > ${outputPrefix}/${outputMain}.${outputSuffix}
+        echo ${outputMain} > someFile.txt
     }
 
     output {
+        File f = read_string("someFile.txt")
```

Comment:
NB this only works because `Upside` is already being delocalized already in another task.

Reply:
:man_facepalming: definitely the wrong test...its been adjusted!